crates/epaint/src/shape.rs

```diff
@@ -356,48 +356,59 @@ impl Shape {
     }
 
     /// Move the shape by this many points, in-place.
-    pub fn translate(&mut self, delta: Vec2) {
+    pub fn transform(&mut self, transform: TSTransform) {
```

Comment:
This does not change the stroke-width of any of the shapes.

Was this intentional?

In any case: please document this.

Reply:
This was not intentional, it was more so I hadn't noticed in the demo anything strange. Upon closer inspection, there are some weird visuals when the stroke width is not changed. Here's a before and after. I'll push the commit adding stroke width scaling, but feel free to revert it! 


https://github.com/emilk/egui/assets/70821802/68f94bcb-b6dd-48d7-b4f0-3b09535e2110


https://github.com/emilk/egui/assets/70821802/a7c46e9b-0519-4061-baa5-e05d239de0ef